src/components/layouts/admin-layout.js

```diff
@@ -164,15 +164,29 @@ const AdminLayout = ({ children }) => {
 				<div className="absolute">
 					<header className="fixed navbar bg-white min-w-full">
 						<p className="px-4 ml-60">{title(entityName)}</p>
-						<div>
-							<Link className="px-4" to={'/'}>
-								Home
-							</Link>
+						<div className="flex justify-center items-center">
 							{entityName && (
 								<Link className="py-2 px-4 mr-4 text-center btn" to={`/${workspaceId}/${entityName}/create`}>
 									Add new
 								</Link>
 							)}
+							<Link className="px-2" to={'/'}>
+								Home
+							</Link>
+							<Link className="px-2" to={`/${workspaceId}/profile`}>

```

Comment:
sharedworkspaceId also must add yes?

Reply:
nope, i check it there is no problem and its work.

Reply:
if workspace has sharedworkspace ,is it also working?

Reply:
Great !